components/global/E2EOnlyBadge.vue

```diff
@@ -9,5 +9,7 @@ export default {
 </script>
 
 <template>
```

Comment:
I originally made this a path prop to pass into the `<Badge />` component but there are edge cases to account for with conditional CSS, etc that felt like a lot since we only need this on these 2 test type badges atm.